src/components/Form/GenericFormCard.tsx

```diff
@@ -1,11 +1,13 @@
+"use client";
+
 import React from "react";
 import { FormSubheading, FormText, StyledCard } from "@/components/Form/formStyling";
 import styled from "styled-components";
 
 interface Props {
     title: string;
     required: boolean;
-    children: React.ReactElement;
+    children: React.ReactNode;
```

Comment:
children should usually be optional, not sure if that's what you want here

Reply:
yeah i think i want there to always be a child :)